src/content/books/lost-approachable-open-source.md

```diff
@@ -0,0 +1,11 @@
+---
+collection: lost
+number: 0
```

Comment:
number 0 is sorta convenient here cause it literally doesn't render, it must be falsey?

Reply:
The pedant in me wants to fix this at the schema level (allowing number to be missing) but this works too. 😄